framework/src/st_things/things_stack/utils/things_network.c

```diff
@@ -237,6 +239,19 @@ int things_set_ap_connection(access_point_info_s *p_info)
 	return 1;
 }
 
+int things_get_ap_list(access_point_info_s*** p_info, int* p_count)
```

Comment:
Should't it be a double pointer?
Current: access_point_info_s*** p_info
Expected: access_point_info_s** p_info

Reply:
fixed.